docs/config.rst

```diff
@@ -18,7 +18,7 @@ Sample Configuration
             document_managers:
                 default:
                     mappings:
-                        AcmeDemoBundle: ~
+                        App: ~
```

Comment:
It would be better to show the standard configuration that is setup by flex: https://github.com/symfony/recipes/blob/main/doctrine/doctrine-bundle/2.9/config/packages/doctrine.yaml

> In AbstractDoctrineExtension.php line 219:                                                                                                  
>  Mapping definitions for Doctrine manager "default" require at least the "type", "dir" and "prefix" options.                                                                                       
                                                                                                    

```suggestion
                        App:
                            is_bundle: false
                            dir: '%kernel.project_dir%/src/Entity'
                            prefix: 'App\Entity'
                            alias: App
```